client/components/tinymce/plugins/wpcom-view/wpvideo-view.jsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import shortcodeUtils from 'lib/shortcode';
+import Shortcode from 'components/shortcode';
+
+var WpVideoView = React.createClass( {
```

Comment:
When writing with ES6 syntax, you shouldn't ever need to use `var`. Prefer `const` or `let` instead.